js/modules/data.js

```diff
@@ -663,39 +663,30 @@ async function removeAllSessions(id) {
 function setifyProperty(data, propertyName) {
   if (!data) return data;
   const returnData = data;
-  if (returnData[propertyName]) {
+  if (returnData[propertyName] && Array.isArray(returnData[propertyName])) {
```

Comment:
I think the first check might be redundant.
```suggestion
  if (Array.isArray(returnData[propertyName])) {
```